python/src/aiconfig/Config.py

```diff
@@ -84,18 +85,37 @@ def create(
         )
 
     @classmethod
-    def load(cls, json_config_filepath) -> "AIConfigRuntime":
+    def load(cls, json_config_filepath: str) -> "AIConfigRuntime":
         """
         Constructs AIConfigRuntime from a JSON file given its file path and returns it.
 
         Args:
             json_config_filepath (str): The file path to the JSON configuration file.
```

Comment:
nit, could call this `config_filepath` and make it clearer that it can be YAML or JSON now